src/typeDefs/mutations.ts

```diff
@@ -111,10 +111,16 @@ export const mutations = gql`
 
     createSampleOrganization: Boolean! @auth
 
+    createVenue(data: VenueInput!, file: String): Venue! @auth
+
     deleteAdvertisementById(id: ID!): DeletePayload!
 
     deleteDonationById(id: ID!): DeletePayload!
 
+    deleteVenue(venueId: ID!, organizationId: ID!): Venue @auth
```

Comment:
No need for `organizationId` here.

Reply:
`organizationId` is being used here to check here if the user who want to delete the venue is admin of the organization or not. Should I take any other approach?